python/plugins/db_manager/dlg_import_vector.py

```diff
@@ -156,9 +160,16 @@ def chooseInputFile(self):
         settings.setValue("/db_manager/lastUsedDir", QFileInfo(filename).filePath())
         settings.setValue("/UI/lastVectorFileFilter", lastVectorFormat)
 
-        self.cboInputLayer.setCurrentIndex(-1)
         self.cboInputLayer.setEditText(filename)
 
+    def inputPathChanged(self, path):
```

Comment:
Can you comment this function?

Reply:
Actually this function wasn't created by me. I used the dlg_import_vector.py from the db_manager plugin installed with QGIS.
After checking the github reference and noticing that in fact it didn't exists. i've remove this function and set as git hub. 
No issues detected in the plugin then.